var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -253,17 +254,20 @@ class PyTorch(PythonPackage, CudaPackage):
 
     # Use patches from IBM's Open CE to enable building on Power systems
     # 03xx - patch temporary to fix a problem that when fixed upstream can be removed
-    patch('https://github.com/open-ce/pytorch-feedstock/raw/main/recipe/0302-cpp-extension.patch',
-          sha256='ecb3973fa7d0f4c8f8ae40433f3ca5622d730a7b16f6cb63325d1e95baff8aa2', when='@1.10.0: arch=ppc64le:')
+    patch('https://raw.githubusercontent.com/open-ce/pytorch-feedstock/0a145060ca8523314ec3893af935c3b140e2d0b0/pytorch-1.10/recipe/0302-cpp-extension.patch',
+          sha256='ecb3973fa7d0f4c8f8ae40433f3ca5622d730a7b16f6cb63325d1e95baff8aa2', when='@1.10.0:1.11.0 arch=ppc64le:')
```

Comment:
This probably isn't going to be fixed in 1.11.1 (if it ever comes out). Would be safer to use `when='@1.10:1.11 arch=...'`

Reply:
That's fair.  I was really just gating it from affecting 1.12.0, which has a separate patch for the same thing, but with a different hash.